README.md

```diff
@@ -5,35 +5,55 @@ This repo contains code used to run the Monash Human Power camera system. The `m
+    Optionally, you may set the overlay to run when the physical switch on the display is toggled by modifying `config.json`. An example of this file's contents could be `{ "activeOverlay": "overlay_all_stats.py" }`.
 
-With that done, dependencies may be installed using `poetry install --dev`. You may then start a shell with `poetry shell`.
+3. **Install Python 3.7 with `pyenv`**
+
+    At this point in time, you'll also need Python 3.7 installed, as currently, the OpenCV version we are using does not have a build available for Python 3.8. [pyenv](https://github.com/pyenv/pyenv) may be used to quickly switch between python versions. To do this,
+
+    - Install `pyenv`.
+        - Linux installation (including Pi) [here](https://github.com/pyenv/pyenv-installer). You may need to install these dependencies from [here](https://github.com/pyenv/pyenv/wiki/Common-build-problems) (use Debian dependencies fo the Raspbery Pi)
+        - macOS installation: See [pyenv's github](https://github.com/pyenv/pyenv)
+        - Windows installation: See [pyenv-win](https://github.com/pyenv-win/pyenv-win)
+    - Install Python 3.7 with `pyenv install 3.7.7`,
+    - and finally, set it as your current python version with `pyenv local 3.7.7`.
+
+4. **Install OpenCV dependencies**
+
+    You will likely need to install some dependencies for OpenCV to run, especially if you are on a Raspberry Pi. To do this, run
+    ```bash
+    sudo apt install libgtk-3-0 libavformat58 libtiff5 libcairo2 libqt4-test libpango-1.0-0 libopenexr23 libavcodec58 libilmbase23 libatk1.0-0 libpangocairo-1.0-0 libwebp6 libqtgui4 libavutil56 libjasper1 libqtcore4 libcairo-gobject2 libswscale5 libgdk-pixbuf2.0-0
+    ```
+
+5. **Install poetry and python dependencies**
+
+    To install dependencies, you will need [poetry](https://python-poetry.org/docs/#installation) installed on your computer.
+
+    With that done, `raspicam` dependencies may be installed using `poetry install --dev`.
 
 ## Usage
 
-*You must be in a poetry shell to run any scripts/tests (see above).*
+**Important:** *You must be in a poetry shell to run any scripts/tests.* To start a poetry shell, run `poetry shell`. You may exit the shell at any time with `exit`.
 
 ### Overlay development
 
-Any overlay (e.g. `overlay_top_strip.py`) may be run from the terminal with Python. You will need a computer with a webcam. The `--host` argument can be used to specify an MQTT broker.
+Any overlay (e.g. `overlay_top_strip.py`) may be run from the terminal with Python. If your computer does not have a webcam, you may replace the video feed with a static background specified with the `--bg` option. The `--host` argument can be used to specify an MQTT broker.
 
 ### Running on Raspberry Pi displays
 
+If you are trying run an overlay over VNC, you will need to enable direct capture mode on the Pi's VNC server. Right click on the VNC logo in the Pi's system tray, click Options -> Troubleshooting -> Enable direct capture mode -> OK.
```

Comment:
I swear i submitted this too. `Options -> Troubleshooting -> Enable`

Reply:
\`Options -> Troubleshooting -> Enable\`  - backslashes work